demisto_sdk/commands/content_graph/parsers/yaml_content_item.py

```diff
@@ -35,7 +35,7 @@ def deprecated(self) -> bool:
 
     @property
     def description(self) -> Optional[str]:
-        return self.yml_data.get('description')
+        return self.yml_data.get('description', '').replace('\\', '')
```

Comment:
Isn't backslash a newline?
If it is - Need to replace it with a single space, right?

Reply:
it's allowing a line break in the file but it's not interpreted as a new line.
<img width="670" alt="image" src="https://user-images.githubusercontent.com/38749041/190893816-5c254510-bdf2-4578-a261-af2a00e9169f.png">
It's causing issues in the CSVs so I removed it. But I'll check if there's a better way to handle this.

Reply:
I'd suggest replacing the backslash with a space.
After all the replacements, replace consecutive spaces with a single space.